internal/hud/hud.go

```diff
@@ -81,6 +81,8 @@ func (h *Hud) Run(ctx context.Context, st *store.Store, refreshRate time.Duratio
 			if err != nil {
 				return err
 			}
+		case <-a.winchCh:
```

Comment:
As discussed aloud, tcell.NewScreenFromTty lets you pass in a chan for winch so you get it as an event.